RiotSwiftUI/Modules/Room/LiveLocationSharingViewer/LiveLocationSharingViewerViewModel.swift

```diff
@@ -29,9 +29,11 @@ class LiveLocationSharingViewerViewModel: LiveLocationSharingViewerViewModelType
 
     // MARK: Private
 
-    private let liveLocationSharingViewerService: LiveLocationSharingViewerServiceProtocol
+    private var liveLocationSharingViewerService: LiveLocationSharingViewerServiceProtocol
```

Comment:
I might have missed it, but does this get updated anywhere?

Reply:
It's required for assigning completion closure.